.github/workflows/build.yml

```diff
@@ -2,119 +2,120 @@
 #   - invoked on push, pull_request, manual trigger, or schedule
 #   - test under at least 3 versions of python
 # -----------------------------------------------------------------------------
-name: build
```

Comment:
This file has not been changed. Fore some reason it is showing up this way after I rebased due to closing the other PR for reusable workflows.